plugins/roleplay.py

```diff
@@ -0,0 +1,317 @@
+                    await respond(msg, "**AFFIRMATIVE. Roles added to race list.**")
+                else:
+                    raise CommandSyntaxError("No roles mentioned.")
+            elif args[1].lower() == "remove":
+                if msg.role_mentions:
+                    for role in msg.role_mentions:
+                        if role.id not in self.plugin_config[gid]["race_roles"]:
+                            self.plugin_config[gid]["race_roles"].remove(role.id)
+                    await respond(msg, "**AFFIRMATIVE. Roles removed from race list.**")
+                else:
+                    raise CommandSyntaxError("No roles mentioned.")
+            else:
+                raise CommandSyntaxError(f"Unsupported mode {args[1].lower()}.")
+
+    @Command("bio",
+             doc="Adds, edits, prints, dumps or deletes character bios.\n"
+                 "Each character name must be unique.\n"
+                 "Fields: race/gender/height/age: limit 64 characters. theme/link: must be viable http(s) url. "
+                 "appearance/equipment/skills/personality/backstory/interests: limit 1024 characters.\n"
+                 "Setting 'race' to the same name as a registered character role will fetch the colour.\n"
+                 "Be aware that the total length of the bio must not exceed 6000 characters.",
+             syntax="\nediting/creating : (name) set (field) [value]\n"
+                    "printing : (name)\n"
+                    "dumping : (name) dump\n"
+                    "deleting: (name) delete",
+             category="role_play",
+             run_anywhere=True)
+    async def _bio(self, msg):
+        """
+        multipurpose command.
+
+        !bio (name) - print out bio
+        !bio (name) (set) (field) [value] - set a bio value
+        !bio (name) (delete) - delete the bio
+
+        :param msg:
+        :return:
+        """
+        gid = str(msg.guild.id)
+        self._initialize(gid)
+        try:
+            args = shlex.split(msg.content)
+        except ValueError as e:
+            self.logger.warning("Unable to split {data.content}. {e}")
+            raise CommandSyntaxError(e)
+
+        if len(args) < 2:
+            raise CommandSyntaxError("At least one argument required")
+
+        t_name = args[1].lower()
+
+        if len(args) == 2:
+            if t_name in self.bios[gid]:
+                await respond(msg, None, embed=self._print_bio(msg.guild, t_name))
+            else:
+                raise CommandSyntaxError(f"No such character {args[1]}")
+        elif len(args) == 3:
+            if args[2].lower() == "delete":
+                if t_name in self.bios[gid]:
+                    if self.bios[gid][t_name].get("author", 0) == msg.author.id or \
+                            msg.author.permissions_in(msg.channel).manage_messages or \
+                                    msg.author.id in self.config_manager.config.get("bot_maintainers", []):
```

Comment:
I don't think this needs a bot maintainer override.

Reply:
Everything needs a bot maintainer override.

Reply:
If that's the case, then add the override to the permission check instead